src/components/rocm/rocm.c

```diff
@@ -261,7 +261,7 @@ rocm_shutdown_component(void)
     }
 
   fn_exit:
-    SUBDBG("EXIT: %s\n", PAPI_strerror(papi_errno));
+    SUBDBG("EXIT\n");
```

Comment:
This is a suggestion and if you don't think it is necessary then go ahead and merge it, but you could add the error code with the SUBDBG e.g. `SUBDBG("EXIT: %d\n", papi_errno)` or `SUBDBG("EXIT: result: %d\n", papi_errno)`.

 `rocd_shutdown` from what I saw only returns `PAPI_OK`, but maybe in the future another error code is added.